container/libcontainer/handler_test.go

```diff
@@ -87,6 +87,8 @@ func TestScanUDPStats(t *testing.T) {
 	if stats != udpstats {
 		t.Errorf("Expected %#v, got %#v", udpstats, stats)
 	}
+
+	r.Close()
```

Comment:
same as above.  Please `defer r.Close()` after we handle the error from os.Open()